root/layout/components/ExternalLinks.js

```diff
@@ -52,6 +55,18 @@ const ExternalLink = ({
     </a>
   );
 
+  if (nonEmpty(entityCredit)) {
+    element = (
+      <>
+        {element}
+        {' '}
+        {bracketedText(
+          texp.l('as {credited_name}', {credited_name: entityCredit}),
+        )}
+      </>
+    );
+  }
```

Comment:
It's not really a role, so I wasn't sure - maybe it would make sense to make this at least `{url} (as {credited_name})` though, in case some languages would want to change the order.

Reply:
I like your suggestion (`'as {credited_name}'` on its own might not be enough context).